Use of undeclared type or module `nasm_rs`

Trying to build rav1e in MSYS2 / MinGW using [media-autobuild_suite](https://github.com/jb-alvarado/media-autobuild_suite/) fails:

Archived log files: [logs.zip](https://0x0.st/sELQ.zip)

**rav1e-git/ab-suite.build.log**
```
CFLAGS: -mthreads -mtune=generic -O2 -pipe
LDFLAGS: -pipe -static-libgcc -static-libstdc++
/opt/cargo/bin/cargo.exe build --release --target=i686-pc-windows-gnu --jobs=4
    Updating git repository `https://github.com/tdaede/nasm-rs.git`
   Compiling rav1e v0.1.0 (file:///H:/development/media-autobuild_suite-master/build/rav1e-git)
error[E0433]: failed to resolve. Use of undeclared type or module `nasm_rs`
  --> build.rs:32:9
   |
32 |         nasm_rs::compile_library_args("rav1easm", &["src/x86/mc.asm"], &[&config_include_arg, "-Isrc/"]);
   |         ^^^^^^^ Use of undeclared type or module `nasm_rs`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0433`.
error: Could not compile `rav1e`.

To learn more, run the command again with --verbose.
```

--- a/build.rs
+++ b/build.rs
@@ -11,12 +11,12 @@ extern crate nasm_rs;
 
 use std::env;
 use std::fs;
-use std::fs::File;
-use std::io::Write;
 use std::path::Path;
 
 fn main() {
-    if cfg!(target_arch = "x86_64") {
+    #[cfg(target_arch = "x86_64")] {
+        use std::fs::File;
+        use std::io::Write;
         let out_dir = env::var("OUT_DIR").unwrap();
         {
             let dest_path = Path::new(&out_dir).join("config.asm");

